wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -42,10 +42,52 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
 
 - [Livestream](https://live.bilibili.com/5333)
 - [Tournament QQ Group](https://jq.qq.com/?_wv=1027&k=5sTdKLp)
+- [challonge](https://challonge.com/zh_CN/cjb4)
+
+## Match schedule
+
+### Round of 16
+
+Saturday，21. July 2018
+
+| Match Time (UTC+8) | Player 1 | | | Player 2 | Match ID |
+| :--: | ---: | ---: | :--- | :--- | :--: |
+| 14:00 | FDchuxin | ![][flag_CN] | ![][flag_CN] | nickawm | 5 |
+| 15:00 | xyxyxy091 | ![][flag_CN] | ![][flag_CN] | Jazzsoldiers | 1 |
+| 16:00 | macronj | ![][flag_CN] | ![][flag_CN] | Crystal | 6 |
+| 20:00 | Skyshifter | ![][flag_CN] | ![][flag_CN] | TC582461 | 2 |
+
+Sunday，22. July 2018
+
+| Match Time (UTC+8) | Player 1 | | | Player 2 | Match ID |
+| :--: | ---: | ---: | :--- | :--- | :--: |
+| 14:00 | tuziel | ![][flag_CN] | ![][flag_CN] | Toki 0w0 | 4 |
+| 15:00 | Snowkyou | ![][flag_CN] | ![][flag_CN] | IA daisuki | 3 |
+| 16:00 | Zhou_Kai | ![][flag_TW] | ![][flag_CN] | Shindou Ayane | 7 |
+| 20:00 | crazybrother | ![][flag_CN] | ![][flag_CN] | williammak | 8 |
 
 ## Mappool
 
-Coming Soon...
+### Round of 16
+
+[Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA)  
```

Comment:
I would use a single line mappacks like this (also add the size of the mappack replacing the XX.X MB part):
**Download the mappack here! (XX.X MB) - [Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA) - [Google Drive](https://drive.google.com/file/d/1ZNekV5Lh6pw3qgl9BuWEs6CN6dSpOl8N)**

`**Download the mappack here! (XX.X MB) - [Baidu Netdisk](https://pan.baidu.com/s/1eSHYDVS7u53S3gzmFK71oA) - [Google Drive](https://drive.google.com/file/d/1ZNekV5Lh6pw3qgl9BuWEs6CN6dSpOl8N)**`